runtime/lua/vim/lsp/health.lua

```diff
@@ -187,27 +187,31 @@ local function check_enabled_configs()
     local config = vim.lsp.config[name]
     local text = {} --- @type string[]
     text[#text + 1] = ('%s:'):format(name)
-    for k, v in
-      vim.spairs(config --[[@as table<string,any>]])
-    do
-      local v_str --- @type string?
-      if k == 'name' then
-        v_str = nil
-      elseif k == 'filetypes' or k == 'root_markers' then
-        v_str = table.concat(v, ', ')
-      elseif type(v) == 'function' then
-        v_str = func_tostring(v)
-      else
-        v_str = vim.inspect(v, { newline = '\n  ' })
-      end
+    if config then
```

Comment:
nit: put the error handling first, especially if it's short compared to the main logic.
```suggestion
    if not config then
      report_warn(...)
```

Reply:
That makes sense! Thanks for the review!